src/code/audio_synthesis.c

```diff
@@ -175,7 +172,7 @@ Acmd* AudioSynth_Update(Acmd* cmdStart, s32* cmdCnt, s16* aiStart, s32 aiBufLen)
         cmdP = AudioSynth_DoOneAudioUpdate(aiBufP, chunkLen, cmdP,
                                            gAudioContext.audioBufferParameters.updatesPerFrame - i);
         aiBufLen -= chunkLen;
-        aiBufP += chunkLen * 2;
+        aiBufP += chunkLen * SAMPLE_SIZE;
```

Comment:
isn't this 2 the "stereo" 2 ? the "sample size 2" seems to already be accounted for by aiBufP being `s16*` + pointer arithmetics